### DISCUSSION TITLE
Integrate co-tracker

### DISCUSSION THREAD

**Discussion by nachonacho2** [talmolab/sleap#2003] [Ideas]
Would it be possible to integrate co-tracker (https://github.com/facebookresearch/co-tracker) as an alternative model for inference, within the SLEAP GUI?

It could be useful to provide initial predictions for further refinement. Or possibly those predictions may be adequate for some use-cases but being able to use the SLEAP interface for interaction with that package would be helpful to many.

**Comment by eberrigan** [comment]
Hi @nachonacho2, 

Thank you for your suggestion. We will see what we can do.

Best,

Elizabeth

**Comment by talmo** [comment]
Just to circle back here: we'd love to integrate these dense trackers (as well as other types of foundation models), but we're currently not planning on adding these to the TensorFlow backend.

We're transitioning to a PyTorch-based backend over in [`sleap-nn`](https://github.com/talmolab/sleap-nn) where I've opened an issue to track these requests: https://github.com/talmolab/sleap-nn/issues/111

We'll definitely be keeping use cases like these in mind as we continue the development! Feel free to let us know if you have more ideas :)

Cheers,

Talmo
